sdks/go/pkg/beam/core/metrics/metrics.go

```diff
@@ -843,13 +872,16 @@ func ResultsExtractor(ctx context.Context) Results {
 			committed[key] = GaugeValue{opt.(*gauge).v, opt.(*gauge).t}
 			r.gauges = append(r.gauges, MergeGauges(attempted, committed)...)
 		case *executionState:
-			for _, v := range opt.(*executionState).state {
-				attempted := make(map[StepKey]MsecValue)
-				committed := make(map[StepKey]MsecValue)
-				attempted[key] = MsecValue{}
-				committed[key] = MsecValue{int64(v.TotalTime)}
-				r.msecs = append(r.msecs, MergeMsecs(attempted, committed)...)
+			attempted := make(map[StepKey]MsecValue)
+			committed := make(map[StepKey]MsecValue)
+			attempted[key] = MsecValue{}
+			committed[key] = MsecValue{[4]time.Duration{0, 0, 0, 0}}
```

Comment:
Nit: New allocations for Go values are always their default 0 value. So `MsecValue{}` and  `MsecValue{[4]time.Duration{0, 0, 0, 0}}` are identical.